openshift_scalability/content/pod-stress-simple.json

```diff
@@ -0,0 +1,37 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "stress-pod",
+    "creationTimestamp": null,
+    "labels": {
+      "name": "stress"
+    }
+  },
+  "spec": {
+    "containers": [
+      {
+        "name": "stress",
+        "image": "lorel/docker-stress-ng",
+        "ports": [
+          {
+            "containerPort": 8080,
+            "protocol": "TCP"
+          }
+        ],
+	"command": ["stress-ng", "--cpu", "1", "--cpu-load", "5", "--vm", "1"],
```

Comment:
Can we please make the command a parameter and have the config file set the default command.   See https://github.com/openshift/svt/blob/master/openshift_scalability/content/logtest/logtest-rc.json#L81 and https://github.com/openshift/svt/blob/master/openshift_scalability/config/ocp-logtest.yaml#L11 for an example

Reply:
Ok, we can leave it as a hard-coded simple pod.   Followup might be to make it a template for a deployment or replication controller in the future.